manifests/s/SyncTERM/SyncTERM/1.1/SyncTERM.SyncTERM.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: SyncTERM.SyncTERM
+PackageVersion: 1.1
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://sourceforge.net/projects/syncterm/files/syncterm/syncterm-1.1/SyncTERM-1.1-setup.exe

```

Comment:
```suggestion
  InstallerUrl: https://cfhcable.dl.sourceforge.net/project/syncterm/syncterm/syncterm-1.1/SyncTERM-1.1-setup.exe
```

Reply:
Why forcing a certain mirror instead of letting SourceForge servers autoselect one based on GeoIP?